tests/AssertTest.php

```diff
@@ -240,6 +240,13 @@ public function getTests()
             array('notRegex', array('abcd', '{^ab}'), false),
             array('notRegex', array('abcd', '{^bc}'), true),
             array('notRegex', array('', '{^bc}'), true),
+            array('letter', array('abcd'), true),
+            array('letter', array('ᴁڅਘธブ'), true),
+            array('letter', array('ȁȄ'), true),
+            array('letter', array('ȁ1Ȅ'), false),
+            array('letter', array('©'), false),
+            array('letter', array('🙁'), false),
+            array('letter', array(''), false),
```

Comment:
How about adding a test including chinese/japanese characters, to show that it is the intended behaviour that these do not pass the assertion.

Reply:
There is a Japanese character included in the test on line 245.

As this checks against the Unicode letter class it will allow Chinese and Japanese characters. It includes all upper and lower case letters from supported alphabets.